app/controllers/articles_controller.rb

```diff
@@ -1,8 +1,12 @@
 class ArticlesController < ApplicationController
- http_basic_authenticate_with name: "dhh", password: "secret", except: [:index, :show]
- 
-	def index
+ #http_basic_authenticate_with name: "dhh", password: "secret", except: [:index, :show]
+ before_action :authenticate_user!
+
+ 	def index
     @articles = Article.all
+    @articles.each do |art|
+      art.text=current_user
```

Comment:
@epintos I used rubocop in a later branch (dev_test) Should I apply it in this one as well? or should I wait to keep the "dev_test" code standards until I merge?
I updated the text in order to trace the scope of "current_user", but not for any practical reason. I will remove it.